Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Landscape Design of South Seas

## Changelog:
- Reworked the landscape of the South Seas. Now there is more land and less sea if you get what I mean. These lands are wastelands and will be for some time.
- Ulduar isn't a wasteland, ruled by a placeholder giant now.


## Scripting changelog (ignore if you can't read CK2 script):
- `hozen` culture becomes `jungle_hozen`.


# How to test:
Check that I didn't break anything. Try to play some time, save, load saves etc.
